### PR TITLE
Delete player from bot.players on leave

### LINKED ISSUE #280: Players list just keeps growing.
It seems like the player list just keeps getting populated but when a user leaves, it is not removed from the players list map. Is it supposed to be like that or is there a possible solution?

### PR DESCRIPTION
**Pull request by AlexKvazos**
Fixes #280

### CODE REVIEW

File: lib/plugins/entities.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ var mobTypes = {
 };
 
 function inject(bot) {
-  
+
```
Comment (AlexKvazos) [inline lib/plugins/entities.js]: This is just whitespace that my text editor removes when I save any file.

**Comment by rom1504** [comment]
Oh I didn't see this pull request, and I just fixed it in 39ce38be13677be8264ade2547501fc4d75471c8

**Comment by AlexKvazos** [comment]
Nice, thanks. Quick question, what version should I be using on my package.json so that it pulls the latest mineflayer from this repo and master branch?

**Comment by rom1504** [comment]
You can put "git://github.com/andrewrk/mineflayer.git#master" instead of the version number in your package.json.

**Comment by rom1504** [comment]
@AlexKvazos and I just release 1.1.1 so you can use that ;)

**Comment by AlexKvazos** [comment]
Thank you!

**Pull request closed without merging**